projects/js-packages/publicize-components/src/social-store/controls.js

```diff
@@ -92,23 +92,31 @@ export default {
 		} );
 	},
 	[ FETCH_SOCIAL_IMAGE_GENERATOR_SETTINGS ]: function () {
-		return apiFetch( { path: '/jetpack/v4/social-image-generator/settings' } );
+		return apiFetch( {
+			path: '/wp/v2/settings?_fields=jetpack_social_image_generator_settings',
+		} );
 	},
 	[ UPDATE_SOCIAL_IMAGE_GENERATOR_SETTINGS ]: function ( action ) {
 		return apiFetch( {
-			path: '/jetpack/v4/social-image-generator/settings',
+			path: '/wp/v2/settings',
 			method: 'POST',
-			data: action.settings,
+			data: {
+				jetpack_social_image_generator_settings: action.settings,
+			},
 		} );
 	},
 	[ FETCH_AUTO_CONVERSION_SETTINGS ]: function () {
-		return apiFetch( { path: '/jetpack/v4/auto-conversion/settings' } );
+		return apiFetch( {
+			path: '/wp/v2/settings?_fields=jetpack_social_autoconvert_images',
+		} );
 	},
 	[ UPDATE_AUTO_CONVERSION_SETTINGS ]: function ( action ) {
 		return apiFetch( {
-			path: '/jetpack/v4/auto-conversion/settings',
+			path: '/wp/v2/settings',
 			method: 'POST',
-			data: action.settings,
```

Comment:
Why do we need multiple API calls? Can we make a single API call to get all the settings at once?

Reply:
We can add the settings fields separated by comma

`/wp/v2/settings?_fields=jetpack_social_image_generator_settings,jetpack_social_autoconvert_images`

Reply:
Yes, to fetch the settings we can just have one call.

Reply:
Yeah, I'll do that.

Reply:
This turned out to be a bigger change as I had to remove the old refreshes introduced in #33969, so I created another PR on this branch for this #34142